lldb/source/API/SBProgress.cpp

```diff
@@ -0,0 +1,34 @@
+//===-- SBProgress.cpp -------------------------------------------*- C++
+//-*-===//
```

Comment:
Broken ASCII art. Also new headers [shouldn't](https://discourse.llvm.org/t/is-c-in-header-files-still-relevant/83124) contain `- C++ ` anymore.